### PR TITLE
AER-1379 add single scenario type

### PR DESCRIPTION
**Pull request by Gijsvanhorn**
and add min and max situation validation information

### CODE REVIEW

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java [hunk 1]
```diff
@@ -59,24 +63,41 @@ public enum CalculationJobType {
...
+  private final int maximumNumberOfSituations;
+
   CalculationJobType(final Set<SituationType> requiredSituationTypes, final Set<SituationType> optionalSituationTypes,
-      final Set<SituationType> multipleSituationsAllowed) {
+      final Set<SituationType> multipleSituationsAllowed, final int minimumNumberOfSituations, final int maximumNumberOfSituations) {
     this.requiredSituationTypes = requiredSituationTypes;
     this.optionalSituationTypes = optionalSituationTypes;
     this.pluralSituationTypes = multipleSituationsAllowed;
+    this.minimumNumberOfSituations = minimumNumberOfSituations;
+    this.maximumNumberOfSituations = maximumNumberOfSituations;
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java:100]: Not sure what the point of these are, and if it makes sense to keep them in this enum: minimum is always 1, and maximum isn't as strict for things like `IN_COMBINATION_PROCESS_CONTRIBUTION` is it? Isn't it usually a situation type that should be at most 1 in a calculation job?

**Review by Hilbrand — CHANGES_REQUESTED** [review]
This is not the right approach. The limit of 6 is also just an arbitrary limit made for the user interface, which should no tbe hardcoded in this library. I suggest to close this pr and add implement the checks in connect server. Put them in a separate validation class that checks for these limits.

**Comment by Gijsvanhorn** [comment]
I removed the number of situations from the enum, and am doubting whether to remove the other validation rules as well. This PR is still relevant because it adds the SINGLE_SITUATION job type though.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java [hunk 2]
```diff
@@ -63,10 +65,19 @@ public enum CalculationJobType {
 
   /**
    * Only 1 Reference scenario
-   *
+   * <p>
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java:68]: I don't really mind this, but we don't use these tags that often and it does seem inconsistent with the other javadocs in this enum.

**Review by BertScholten — APPROVED** [review]
LGTM

**Review by Hilbrand — APPROVED** [review]
LGTM. Other validations can be left in.They specify something about the CalculationJobType.

**Pull request merged**